tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+
+@pytest.fixture
+def random_seed():
+    return 42
```

Comment:
This isn't random lol I think you should just rename it to `test_seed` so it's not misleading.

Reply:
Fair point. It's a seed for random values, usually it's called this way. Was quite random after I rolled the dice :-D
I can rename it to default_seed, if you like it better?

Reply:
I would avoid the prefix `test_` it is commonly used as a prefix for tests in python. 

perhaps `random_state` or simply pass `random_state=42` directly in the code below instead of a fixture.

Reply:
I renamed it to "default_seed" to avoid "random" in its name